packages/form-validations/README.md

```diff
@@ -126,4 +191,52 @@ _Get the errors of the form or a specific field_
 | Parameter | Default | Required? | Description                                                                                                                                                                                                                                                            |
 | --------- | ------- | --------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | input     | -       | ✕         | When `input` is given, it returns the errors of that input element, otherwise returns all errors of the form as an object, using input element as key and corresponding errors as value. `validate()` must be called before expecting this method to return correctly. |
+
+```
+
+**FormValidations.setLocale(locale)**
+
+_Set the current locale globally_
+
+| Parameter | Default | Required? | Description                                                                     |
+| --------- | ------- | --------- | ------------------------------------------------------------------------------- |
+| `locale`  | -       | ✔         | Error messages on new FormValidations forms will be displayed according to this locale |
+
+**FormValidations.addMessages(locale, messages)**
+
+_Set the current locale globally_
+
+| Parameter  | Default | Required? | Description                                                         |
+| ---------- | ------- | --------- | ------------------------------------------------------------------- |
+| `locale`   | -       | ✔         | The corresponding locale                                            |
+| `messages` | -       | ✔         | Object containing validator names as keys and error texts as values |
+
+**FormValidations.addValidator(elem, fn, msg, priority, halt)**
+
+_Add a custom validator_
+
+| Parameter | Default  | Required? | Description|
+| ---       | ----     |   -----   | ---        |
+| `elem`| - | ✔ | The dom element where validator is applied to.|
+| `fn`| - | ✔ | The function that validates the field. Value of the input field gets passed as the first parameter, and the attribute value (split using comma) as the subsequent parameters. For example, for `<input data-spirit-my-validator="10,20,dhaka" value="myValue"/>`, validator function get called like `fn("myValue", 10, 20, "dhaka")`. Inside the function `this` refers to the input element|
+| `message`| - | ✔ | The message to show when the validation fails. It supports simple templating. `${0}` for the input's value, `${1}` and so on are for the attribute values. For the above example, `${0}` will get replaced by `myValue`, `${1}` by `10`, `${2}` by `20`, `${3}` by `dhaka`. It can also be a function which should return the error string. The values and inputs are available as function arguments|
```

Comment:
```suggestion
| `message`| - | ✔ | The message to show when the validation fails. It supports simple templating. `${0}` for the input's value, `${1}` and so on are for the attribute values. For the above example, `${0}` will get replaced by `myValue`, `${1}` by `10`, `${2}` by `20`, `${3}` by `dhaka`. It can also be a function which should return the error string. The values and inputs are available as function arguments |
```

Reply:
Fixed.